docs/services/vision/detection.md

```diff
@@ -223,7 +223,7 @@ Wait for the robot to reload, and then go to the **Control** tab to test the str
 
 ## Code
 
-The following code gets the robot’s vision service and then runs a color detector vision model on an image from the robot's camera `"camera_1"`:
+The following code gets the robot’s Vision Service and then runs a color detector vision model on output from the robot's camera `"cam1"`:
```

Comment:
```suggestion
The following code gets the robot’s vision service and then runs a color detector vision model on output from the robot's camera `"cam1"`:
```

I think we settled on capitalizing if it's the abstract service and not if it's an instance of it